tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks/Models/VersionDetails.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Xml.Serialization;
+
+namespace Microsoft.DotNet.SourceBuild.Tasks.Models
+{
+    [XmlRoot("Dependencies")]
+    public class VersionDetails
+    {
+        [XmlArray("ToolsetDependencies")]
+        public Dependency[] ToolsetDependencies { get; set; }
+        [XmlArray("ProductDependencies")]
+        public Dependency[] ProductDependencies { get; set; }
+    }
+
+    public class Dependency
```

Comment:
\<nit> I guess I didn't notice that this class is "extra" in the file vs. `VersionDetails` matching `VersionDetails.cs`. For models I think it's *ok* to include multiple classes in the file, but IMO the filename shouldn't match one of the classes in particular to avoid potentially being a bit misleading. Either way, nitpicking with it being so small anyway.

Reply:
Renamed the file.